net/privoxy/Makefile

```diff
@@ -123,7 +124,7 @@ define Package/privoxy/preinst
 
 # stop service if PKG_UPGRADE
 [ "$${PKG_UPGRADE}" = "1" ] && /etc/init.d/privoxy stop >/dev/null 2>&1
-	
+
```

Comment:
Unrelated change

Reply:
Removed unnecessary spaces and processed them together